src/components/separators/Separator.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const SIZE = {
+  NORMAL: 'normal',
+  SMALL: 'small',
+  LARGE: 'large'
+};
+
+const Separator = ({size = SIZE.NORMAL}) => {
```

Comment:
IMHO better change to SeparatorVertical
and then change styles from `sg-separator` to `sg-vertical-separator` - there is only 6 occurrence in  our projects so we can change it easily and have better described naming
WDYT @aju @kdzwinel @burakukula @matzimowski @vergilius